### DISCUSSION TITLE
Chunks occasionally fail to load through webworker.

### DISCUSSION THREAD

**Discussion by rapidfixer** [vercel/next.js#65791] [Help]
### Summary

Occasionally, we receive a lot of error reports in Sentry with the failure to load some chunks, for example:
`Uncaught NetworkError: Failed to execute 'importScripts' on 'WorkerGlobalScope': The script at 'https://www.website.com/_next/static/chunks/2328.b32ef1a144862ca5.js' failed to load.
`Chunks are different for different error events.

As far as i understand, from some moment loading new chunks was moved to webworker. (Previously that would be 'ChunkLoadError' emitted not from the worker)

Static file exists and available, so error looks like pretty random.
I've tried to use https://github.com/mattlewis92/webpack-retry-chunk-load-plugin to force retry attempts in case if it was really only network problem, but it is not helping a lot.
I see that missing chunk is not leading to hard reload, when artificially simulating unavailable chunk. It also not leading to generation of /error page. I think, it is possible to force hard reload through _retry-chunk-load-plugin_, but it is not the best option.
No issues/discussions found on that matter. How should Next handle this kind of issues?

### Additional information

```js
Next v14.2.3 standalone.
```

**Comment by navanshu** [comment]
We majority of the time get this issue, sometimes they will load and sometimes they won't don't why is nextjs server busy, individually request these files after a few minutes, we are able to get them
